src/Cache/Engine/RedisEngine.php

```diff
@@ -137,6 +137,7 @@ protected function connectRedisCluster(): bool
     {
         $connected = false;
 
+        // @codeCoverageIgnoreStart
```

Comment:
Why do we need this?

Reply:
Codecov doesn't see this
https://github.com/cakephp/cakephp/pull/17982/files#diff-05077d970334dd20377d2fcfdefeda53ed0c9849c172c66c53f8a6cd9a0c75efR203

and I don't know how to make it work. If you have an idea how to make it work then please execute.
TLS can't be easily tested anyways.